src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf8Utility.Transcoding.cs

```diff
@@ -5,8 +5,8 @@
 using System.Buffers;
 using System.Diagnostics;
 using System.Numerics;
-using System.Runtime.CompilerServices;
```

Comment:
This got removed by mistake and caused some of the build failures. Re-adding it.